pycparser/c_generator.py

```diff
@@ -6,19 +6,20 @@
 # Eli Bendersky [https://eli.thegreenplace.net/]
 # License: BSD
 #------------------------------------------------------------------------------
-from . import c_ast
+from . import c_ast, c_parser
 
 
 class CGenerator(object):
     """ Uses the same visitor pattern as c_ast.NodeVisitor, but modified to
         return a value from each visit method, using string accumulation in
         generic_visit.
     """
-    def __init__(self):
+    def __init__(self, flatten=False):
```

Comment:
I'd suggest a more descriptive name for this option. `flatten` can mean meany things.

Something like `minimal_parens`, etc

Reply:
Using `reduce_parentheses`